************* Module pip._vendor.urllib3.util.ssl_
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_vendor/urllib3/util/ssl_.py:19:0: C0103: Constant name "SSLContext" doesn't conform to UPPER_CASE naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_vendor/urllib3/util/ssl_.py:20:0: C0103: Constant name "SSLTransport" doesn't conform to UPPER_CASE naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_vendor/urllib3/util/ssl_.py:30:0: C0103: Argument name "a" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_vendor/urllib3/util/ssl_.py:30:0: C0103: Argument name "b" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_vendor/urllib3/util/ssl_.py:65:4: C0103: Constant name "PROTOCOL_SSLv23" doesn't conform to UPPER_CASE naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_vendor/urllib3/util/ssl_.py:70:8: C0103: Constant name "PROTOCOL_SSLv23" doesn't conform to UPPER_CASE naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_vendor/urllib3/util/ssl_.py:72:8: C0103: Constant name "PROTOCOL_SSLv23" doesn't conform to UPPER_CASE naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_vendor/urllib3/util/ssl_.py:133:4: E0102: class already defined line 19 (function-redefined)
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_vendor/urllib3/util/ssl_.py:133:4: R0205: Class 'SSLContext' inherits from object, can be safely removed from bases in python3 (useless-object-inheritance)
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_vendor/urllib3/util/ssl_.py:133:4: R0902: Too many instance attributes (10/7) (too-many-instance-attributes)
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_vendor/urllib3/util/ssl_.py:179:19: W1505: Using deprecated method wrap_socket() (deprecated-method)
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_vendor/urllib3/util/ssl_.py:161:38: W0613: Unused argument 'server_hostname' (unused-argument)
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_vendor/urllib3/util/ssl_.py:323:8: W0201: Attribute 'post_handshake_auth' defined outside __init__ (attribute-defined-outside-init)
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_vendor/urllib3/util/ssl_.py:350:12: W0201: Attribute 'keylog_filename' defined outside __init__ (attribute-defined-outside-init)
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_vendor/urllib3/util/ssl_.py:355:0: R0913: Too many arguments (13/5) (too-many-arguments)
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_vendor/urllib3/util/ssl_.py:355:0: R0914: Too many local variables (18/15) (too-many-locals)
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_vendor/urllib3/util/ssl_.py:403:8: C0103: Variable name "e" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_vendor/urllib3/util/ssl_.py:404:12: W0707: Consider explicitly re-raising using the 'from' keyword (raise-missing-from)
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_vendor/urllib3/util/ssl_.py:355:0: R0912: Too many branches (13/12) (too-many-branches)
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_vendor/urllib3/util/ssl_.py:472:32: C0103: Variable name "f" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_vendor/urllib3/util/ssl_.py:489:8: W0212: Access to a protected member _validate_ssl_context_for_tls_in_tls of a client class (protected-access)
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_vendor/urllib3/util/ssl_.py:492:4: R1705: Unnecessary "else" after "return" (no-else-return)

-----------------------------------
Your code has been rated at 8.65/10

